src/components/ui/Chat/ChatBubble/index.tsx

```diff
@@ -24,11 +26,15 @@ export interface ChatBubbleProps extends Omit<HTMLAttributes<HTMLDivElement>, 'c
   children?: ReactNode | ReactNode[];
 };
 
-export const ChatBubble: FC<ChatBubbleProps> = ({ variant, senderName, content, showName = true, showTail, redacted, children, ...rest }) => {
+export const ChatBubble: FC<ChatBubbleProps> = ({ variant, senderName, timestamp, content, showName = true, showTail, redacted, children, ...rest }) => {
   return (
     <StyledChatBubble variant={variant} editable={false} {...rest} data-testid='chat-bubble'>
-      {showName && <div className='ch-sender-name'>{senderName}</div>}
-      <div>{content}</div>
+      {showName &&
+        <div className='ch-header'>
+          <div className='ch-sender-name'>{senderName}</div>
+          {timestamp && <div className='ch-header-timestamp'>{timestamp}</div>}
+        </div>}
+      {content && <div>{content}</div>}
```

Comment:
I think if this is part of the breaking change we talked about, we should get rid of `content` all together an only use the `children` prop.